CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 endif ()
 
 set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/build/")
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_BINARY_DIR}/generators/")
```

Comment:
Out of interest: Is there a reason to use `CMAKE_BINARY_DIR` instead of `CMAKE_SOURCE_DIR`?

Reply:
First, we save the `/build/` that was necessary before. (`${CMAKE_SOURCE_DIR}/build/` was always equivalent to `${CMAKE_BINARY_DIR}/build`)

But more importantly,  `CMAKE_BINARY_DIR` is now a lot less _magical_ in the discovery of the dependency files. For example before, CLion used `cmake-build-debug/` as the build folder. Still, `build/` was used to find the dependencies. Now, `CMAKE_BINARY_DIR` will already point at `cmake-build-debug/` and looking for dependencies is relative to this directory